src/3D/Camera.cpp

```diff
@@ -183,9 +199,23 @@ void Camera::handleMouseInput(const SDL_Event& e)
 	}
 }
 
+#define BV(v) (btVector3((v).x, (v).y, (v).z))
+#define AIR_DENSITY (1.2f)
```

Comment:
Use a static const, preferably as part of a namespace.

Reply:
Removed these macros. This was part of an attempt to model camera movement physics in a more realistic manner